app/views/wpcc/your_contributions/_minimum_contribution_error.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="minimum-contributions-error is-hidden">
+  <div class="validation-summary" role="alert">
+    <div class="validation-summary__content-container">
+      <p class="validation-summary__title">
+        <%= t('wpcc.dough.forms.validation.summary.title') %>
```

Comment:
I'm getting an error here, according to the translation file this should be "dough.forms.validation.summary.title" and so I'm getting a missing translation.

Reply:
good spot @andreferreiradlw not sure why it worked for me initially, but all sorted now